CMakeLists.txt

```diff
@@ -78,13 +90,26 @@ else()
     list(GET VERSION_LIST 0 LLVM_VERSION_MAJOR)
     list(GET VERSION_LIST 1 LLVM_VERSION_MINOR)
     list(GET VERSION_LIST 2 LLVM_VERSION_PATCH)
+    find_path(LibClang_INCLUDE_DIR stddef.h HINTS
+      ${LLVMINCLUDEDIR}/clang/${LLVMVERSION}/include
+      ${LLVMINCLUDEDIR}/../clang/${LLVMVERSION}/include
+      ${LLVMINCLUDEDIR}/../../lib/clang/${LLVMVERSION}/include
+      ${LLVMINCLUDEDIR}/../../lib64/clang/${LLVMVERSION}/include
+      ${CLANG_INCLUDE_DIRS}/clang/${LLVMVERSION}/include
+      ${CLANG_INCLUDE_DIRS}
+      )
```

Comment:
again, maybe see if we can iterate over list of `[ clang, ../clang, ../lib/clang/, ../lib64/clang/ ]`